lib/index.js

```diff
@@ -284,21 +315,31 @@ api.get = async ({actor, account, email, type} = {}) => {
  *
  * @return a Promise that resolves once the operation completes.
  */
-api.remove = async ({actor, account, type} = {}) => {
+api.remove = async ({actor, account, type, existingToken} = {}) => {
```

Comment:
Would "existingToken" be better named just "token"?

Reply:
It may end up being just `id` if we add token IDs.

Reply:
This should use the ID of the token now that that's a thing.